test/specs/languages/test-cpp.el

```diff
@@ -0,0 +1,106 @@
+;;; test-cpp.el --- Flycheck Specs: C++      -*- lexical-binding: t; -*-
```

Comment:
We use `c++` to refer to this language all throughout our tests.  Please rename this file to `test-c_c++.el` if possible.